app/services/slot_availability.rb

```diff
@@ -29,7 +29,7 @@ def all_slots
 
 private
 
-  attr_reader :prison, :noms_id, :date_of_birth, :start_date, :end_date
```

Comment:
How come these objects can't be value objects anymore? I don't see any need for them to be mutable.